R/findC.R

```diff
@@ -27,7 +27,8 @@ findC <- function(timepoints, max.lag = NULL, pi = 0.95, iter = 10){
     max.lag <- floor(length(timepoints) / 4)
   }
   #checking through all the conditions
-  stopifnot(all(is.numeric(timepoints)), max.lag <= length(timepoints) / 4, max.lag %% 1 == 0)
+  stopifnot(all(is.numeric(timepoints)), max.lag <= length(timepoints) / 4, max.lag %% 1 == 0,
```

Comment:
This should have the same requirement that `max.lag >= 1` that `bestlag.R` has.  Does `corr_bestlag.R` also have this test for `max.lag`?

Reply:
I can add this to corr_bestlag but wouldn't it be redundant? Because of the function corr_bestlag calls the best.lag function.

Reply:
I made the changes anyway. We had a lot of redundancy.

Reply:
I don't see it as redundant.  Every function that is exported can be called directly be a user, so we need to protect all of the exported functions from invalid input.